lib/modules/manager/gradle-wrapper/utils.ts

```diff
@@ -61,6 +65,29 @@ export function getJavaConstraint(
   return '^11.0.0';
 }
 
+// https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:daemon_jvm_criteria
+export async function getJvmConfiguration(
```

Comment:
This function looks like it always returns a major version, e.g. 11, 17, etc. Is there some other edge case why you didn't return a number instead of a string?

Reply:
No edge case I'd be aware of. The return value of `getJvmConfiguration()` is string-concatenated with `.0.0.0`, so I thought it would be somewhat redundant to use `parseInt()`, return an int and then have it converted back to a string internally.

Reply:
Looks like no particular edge case right now according to: https://docs.gradle.org/current/userguide/gradle_daemon.html#daemon_jvm_criteria
If support for minor version is added in the future (which seems unlikely), this solution would have to be tweaked slightly.
I also suppose JDK vendor is a bit of a stretch goal for this solution.